lib/database.dart

```diff
@@ -23,16 +24,61 @@ class MyDatabase extends ChangeNotifier{
     return _instance;
   }
 
+  final Map<int, Future<void> Function(Database)> _migrations = {
+    1: _migrateFromV1toV2,
+    2: _migrateFromV2toV3,
+  };
+
+  static Future<void> _migrateFromV1toV2(Database db) async {
+    await db.execute('''
+      CREATE TABLE Note_new (
+        id VARCHAR(36) PRIMARY KEY,
+        title TEXT,
+        content TEXT
+      )
+    ''');
+    await db.execute('''
+      INSERT INTO Note_new (id, title, content)
+      SELECT id, title, content
+      FROM Note
+    ''');
+    await db.execute("DROP TABLE Note");
```

Comment:
Is this affect to the current data of user?

Reply:
I have conducted some checks on moving all data to the new table (line 40 - 44), all the fields work fine